main.ts

```diff
@@ -43,7 +43,35 @@ class Translator {
 
   visitParameters(fn: ts.FunctionLikeDeclaration) {
     this.emit('(');
-    this.visitList(fn.parameters);
+    var i;
+    for (i = 0; i < fn.parameters.length; i++) {
```

Comment:
I think you can, because we claim to be targetting ES6, however, we are not calling node with --harmony which you'd need to fix first.